xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/dto/marketdata/response/DeribitOrderBookResponse.java

```diff
@@ -0,0 +1,22 @@
+package org.knowm.xchange.deribit.v2.dto.marketdata.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.knowm.xchange.deribit.v2.dto.DeribitError;
+import org.knowm.xchange.deribit.v2.dto.DeribitResponse;
+import org.knowm.xchange.deribit.v2.dto.marketdata.DeribitOrderBook;
+
+public class DeribitOrderBookResponse extends DeribitResponse<DeribitOrderBook> {
```

Comment:
why not just use DeribitResponse<DeribitOrderBook> directly in the client interface?
these type of classes seams to add nothing usefull

Reply:
Valid point, i think i was driven by other exchange implementation.